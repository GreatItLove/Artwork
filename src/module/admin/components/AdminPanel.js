import React from 'react';
import Header from './Header';
import SidebarMenuRouters from './SidebarMenuRouters';
/**
 * @function Admin
 * @returns {XML}
 */
function Admin() {
  return (
    <div>
      <Header />
      <SidebarMenuRouters />
    </div>
  );
}

export default Admin;
