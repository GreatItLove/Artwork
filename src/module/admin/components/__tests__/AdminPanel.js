/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import AdminPanel from '../AdminPanel';

describe('Admin component', () => {
  it('Shallow snapshot', () => {
    const wrapper = shallow(<AdminPanel />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
