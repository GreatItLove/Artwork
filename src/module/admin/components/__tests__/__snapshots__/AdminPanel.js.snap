// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin component Shallow snapshot 1`] = `
<div>
  <Header />
  <SidebarMenuRouters />
</div>
`;
